tests/api/core/strapi/api/validate/validate-query.test.api.js

```diff
@@ -444,12 +455,12 @@ describe('Core API - Validate', () => {
       });
 
       // TODO: Nested sort returns duplicate results. Add back those tests when the issue will be fixed
-      describe.skip('Relation', () => {
+      describe('Relation', () => {
         describe('Scalar', () => {
-          describe('Basic (no modifiers)', () => {
```

Comment:
Here I'm just adding back a portion of the skipped tests; adding back all of them didn't work unfortunately, so I left the referenced TODO about a bug